docs/use_cases/model_applications/marine_and_cryosphere/UserScript_fcstRTOFS_obsAOML_calcTransport.py

```diff
@@ -0,0 +1,159 @@
+# | This tarball should be unpacked into the directory that you will set the value of INPUT_BASE. See `Running METplus`_ section for more information.
+#
+# | **Data Source:** NOMADS RTOFS Global + Daily mean transport (https://www.aoml.noaa.gov/phod/floridacurrent/data_access.php)+ Eightmilecable (static, provided with the use case)
+
+##############################################################################
+# External Dependencies
+# ---------------------
+#
+# You will need to use a version of Python 3.6+ that has the following packages installed:
+#
+# * scikit-learn
+# * pyproj
+#
+# If the version of Python used to compile MET did not have these libraries at the time of compilation, you will need to add these packages or create a new Python environment with these packages.
+#
+# If this is the case, you will need to set the MET_PYTHON_EXE environment variable to the path of the version of Python you want to use. If you want this version of Python to only apply to this use case, set it in the [user_env_vars] section of a METplus configuration file.::
+#
+#    [user_env_vars]
+#    MET_PYTHON_EXE = /path/to/python/with/required/packages/bin/python
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case utilizes the METplus UserScript wrapper to generate a
+# command to run with Python Embedding for the specified valid time. 
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# This use case uses UserScript. All the gridded data being pulled from the files via Python Embedding. 
+# All of the desired statistics are in the log file.
+# It processes the following run time:
+#
+# | **Valid:** 2021-10-28
+# 
+# The code grabs the 20211028, 20211027, and 20211026 24 hour RTOFS files. 
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# i.e. -c parm/use_cases/model_applications/marine_and_cryosphere/UserScript_fcstRTOFS_obsAOML_calcTransport.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/model_applications/marine_and_cryosphere/UserScript_fcstRTOFS_obsAOML_calcTransport.conf
+
+##############################################################################
+# MET Configuration
+# ---------------------
+#
+# None. All of the processing is completed in the UserScript
+#
+
+##############################################################################
+# Python Embedding
```

Comment:
Not sure what the appropriate title should be for this section, but it is not technically Python Embedding.
```suggestion
# User Script
```